.gitignore

```diff
@@ -0,0 +1,17 @@
+
+.vscode/c_cpp_properties.json
+.vscode/launch.json
+.vscode/settings.json
+.vscode/settings.json
+edweiss-app/package.json
+edweiss-app/package-lock.json
+edweiss-app/package-lock.json
```

Comment:
Just watch out for the package-lock.json as they are needed for the CI jest rule. But beside that it's okay for me.